install/fedora-installation-guide.md

```diff
@@ -3,7 +3,7 @@
 > **Notes:**
 >
 > - Kata Containers packages are available for [Fedora\*](https://fedoraproject.org)
->   versions **26** and **27** (currently `x86_64` only).
+>   versions **26**, **27** and **28** (currently `x86_64` only).
```

Comment:
Shouldn't we remove mention of F26 as it's now EOL?

- https://en.wikipedia.org/wiki/Fedora_version_history
- https://fedoraproject.org/wiki/End_of_life

If so, we'll also need to remove the F26 OBS packages and config too..